### ISSUE TITLE
Add unicode codepoint options for all emojiwords

### ISSUE THREAD

**Issue by vbatts** [emojisum/emojisum#8]
From #5, add Unicode codepoint words for all the current entries.

https://github.com/vbatts/emojisum/blob/master/emoji/emojimap.json
Now supports having multiple options per position in the list, provided they render the same emoji.
Initially we have the colon notation, but would like the unicode codepoint option as well.

**Comment by vbatts** [comment]
http://www.unicode.org/emoji/charts/full-emoji-list.html (thanks @smunilla)

**Comment by vbatts** [comment]
reopen, as they all don't have codepoints yet.

**Comment by vbatts** [comment]
needs `:wine:`

**Comment by smunilla** [comment]
There are actually a few on the list I'm not a huge fan of. I'll probably open another PR addressing that once this is done.

**Comment by vbatts** [comment]
Perhaps switch from `:blue_car:` to `:police_car:`

**Comment by vbatts** [comment]
fixed with #27 !
